### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MarkiMark3**
[DEMO LINK](https://MarkiMark3.github.io/react_list-of-goods-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,45 +16,105 @@ export const goodsFromServer = [
...
+        </button>
+        {reverseField || sortField
+          ? (
+            <button
+              type="button"
+              className="button is-danger is-light"
+              onClick={() => {
+                setReverseField(false);
+                setSortField('');
+              }}
```
Comment (anastasiiavorobiova) [inline src/App.jsx:105]:
```suggestion
              onClick={resetSort}
```
Consider moving this logic from jsx to a separate function

File: src/App.jsx [hunk 2]
```diff
@@ -14,45 +16,105 @@ export const goodsFromServer = [
...
+            >
+              Reset
+            </button>
+          )
+          : null}
+      </div>
+
+      <ul>
+        {visibleGoods.map(good => (
+          <li data-cy="Good">{good}</li>
```
Comment (anastasiiavorobiova) [inline src/App.jsx:115]: Consider using keys for rendering list items

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Added a few comments to fix

File: src/App.jsx (hunk 2 above)
Comment (MarkiMark3) [reply src/App.jsx:115]: I dont have ID for keys in this task, and VS is not letting me to use index as keys, so I am not sure what do you mean please elaborate

**Pull request closed without merging**